### DISCUSSION TITLE
Can job runner handle a load of 5 lakh plus enqueued jobs ?

### DISCUSSION THREAD

**Discussion by vimal-raj467** [jobrunr/jobrunr#1072] [Q&A]
I have to process around 5 lakh jobs. Below is my worker config 

![SS](https://github.com/user-attachments/assets/01b6abee-f88e-4b5b-9247-b67e3f981c99)

<img width="253" alt="Screenshot 2024-07-19 at 10 05 52 PM" src="https://github.com/user-attachments/assets/297098eb-ed2e-4644-a50f-0cba324ad3f1">

Under high load facing the following issues,
1. Server count sporadically becomes zero and processing jobs at stuck at zero
2. Dashboard keeps on loading.

**Comment by rdehuyss** [comment]
Check your db load. More servers = more load on your DB.

In our performance test, we do 500k jobs in about 3 minutes.

**Reply by vimal-raj467** [reply]
We have multiple sql datasources configured, db for jobrunnr is in Azure & our data is in AWS rds. 
Will check the metrics for both. Thanks.
